auth/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,23 @@
             android:theme="@style/FirebaseUI.Transparent"/>
 
         <activity
+            android:name=".provider.GitHubLoginHolder"
+            android:label=""
+            android:launchMode="singleInstance"
```

Comment:
This is the first main issue: adding `singleInstance` creates a new task which very bad. The UI looks weird and if the user goes to their recent apps list, the CCT instance shows up separate from the login app. I need this so that I can get an `onNewIntent` call without a new instance of the holder activity being created. Is there some other way to accomplish this? I've tried `singleTop` with no success.